frontend/src/app/Results/ResultsPageCopr.tsx

```diff
@@ -149,11 +149,7 @@ const ResultsPageCopr = () => {
             <br />
             <List>
               <ListItem>
-                <code>sudo yum install -y dnf-plugins-core</code> on RHEL 8 or
-                CentOS Stream
-              </ListItem>
-              <ListItem>
-                <code>sudo dnf install -y dnf-plugins-core</code> on Fedora
+                <code>sudo dnf install -y dnf-plugins-core</code>
```

Comment:
We could also be more specific, WDYT?

```suggestion
                <code>sudo dnf install -y 'dnf-command(copr)'</code>
```

Reply:
I am not sure, reading [here](https://docs.pagure.org/copr.copr/how_to_enable_repo.html#how-to-enable-repo) I will stick with `sudo dnf install -y dnf-plugins-core`

Reply:
The benefit of using the virtual provide is that if the package is renamed or split into more packages for example it will still work.